_data/other.yml

```
@@ -281,6 +281,16 @@ websites:
       exceptions:
           text: "2FA only for North American customers that create Personal campaigns."
 
+    - name: Grammarly
+      url: https://grammarly.com/
+      img: grammarly.png
+      tfa: Yes
+      sms: Yes
+      software: Yes
+      doc: https://support.grammarly.com/hc/en-us/articles/360032922091
+      exceptions:
+          text: "SMS required for 2FA."
```

This is the text we've used on sites with similar requirements.
```suggestion
          text: "SMS-capable phone is required."
```